crates/redis_interface/src/commands.rs

```diff
@@ -852,12 +852,31 @@ impl super::RedisConnectionPool {
             .await
             .change_context(errors::RedisError::ConsumerGroupClaimFailed)
     }
+
+    #[instrument(level = "DEBUG", skip(self))]
+    pub async fn incr_keys_using_script<V>(
+        &self,
+        lua_script: &str,
+        key: Vec<String>,
```

Comment:
Can't this be multiple keys?

Reply:
Can be but do we also want to make the keys generic,I mean should we allow that? As today in our redis-interface module there's no other key than a string